app/templates/app/image.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
```

Comment:
Weshalb gibt es eine eigene View, um das Buchbild anzuzeigen? Das Bild müsste in der _book_card.html angezeigt werden. Hier einfach das Beispiel-Bild mit der richtigen URL ersetzen.